_format/1.0/index.md

```diff
@@ -240,13 +240,14 @@ alongside other information to be represented in a resource object, these keys
 #### <a href="#document-resource-object-relationships" id="document-resource-object-relationships" class="headerlink"></a> Relationships
 
 The value of the `relationships` key **MUST** be an object (a "relationships
-object"). Members of the relationships object ("relationships") represent
-references from the [resource object][resource objects] in which it's defined to other resource
-objects.
+object"). Members of the relationships object represent "relationships", i.e.,
+references from the [resource object][resource objects] in which they are defined
+to other resource objects.
 
 Relationships may be to-one or to-many.
 
-A "relationship object" **MUST** contain at least one of the following:
+Each relationship **MUST** be represented by an object (a "relationship object")
```

Comment:
One potential problem with "each relationship's contents" is that the relationship object also includes links, and I'm not sure if those are part of the contents???